powersimdata/design/investment/const.py

```diff
@@ -79,6 +79,7 @@
     "LTRG1",  # Single tech for wind
     "4Hr Battery Storage",  # Single tech for storage
     "Seattle",  # Single tech for solar
+    "*",  # Single tech for nuclear
```

Comment:
That is a pretty bad choice for accessing nuclear information.

Reply:
To be clear, this is not wildcard matching. The CSV file literally has an entry `*`.

Reply:
Yeah, it is terrible. I guess there is a documentation but anything else could be more explanatory.

Reply:
There is definitely room for improvement. Currently, we're somewhat-arbitrarily selecting one `TechDetail` to consider for each technology, hard-coded. It would be nice to give the user more flexibility to select different ones, when they are available, or maybe even to define their own.